### ISSUE TITLE
cc and bcc email fields don't work

### ISSUE THREAD

**Issue by twakeford** [ubiquitypress/rua#473]
When using the cc and bcc fields when sending emails, the 'to' recipient receives the email correctly, but neither the cc or bcc recipients receive the email.

= fix so that all email addresses send correctly, regardless of whether in to, cc, or bcc

**Comment by ajrbyers** [comment]
Is this using the email pop-up?

**Comment by twakeford** [comment]
yes - I used the email icon links on the submission details page
